src/typescript/frontend/src/app/page.tsx

```diff
@@ -1,45 +1,6 @@
-import { type HomePageParams, toHomePageParamsWithDefault } from "lib/routes/home-page-params";
-import HomePageComponent from "./home/HomePage";
-import { REVALIDATION_TIME } from "lib/server-env";
-import { headers } from "next/headers";
-import { isUserGeoblocked } from "utils/geolocation";
-import {
-  fetchFeaturedMarket,
-  fetchMarkets,
-  fetchNumRegisteredMarkets,
-  fetchPriceFeed,
-} from "@/queries/home";
-import { symbolBytesToEmojis } from "@sdk/emoji_data";
+"use client";
+import LoadingComponent from "components/loading";
 
-export const revalidate = REVALIDATION_TIME;
-export const dynamic = "force-dynamic";
-
-export default async function Home({ searchParams }: HomePageParams) {
-  const { page, sortBy, orderBy, q } = toHomePageParamsWithDefault(searchParams);
-  const searchEmojis = q ? symbolBytesToEmojis(q).emojis.map((e) => e.emoji) : undefined;
-
-  const numRegisteredMarkets = await fetchNumRegisteredMarkets();
-  const featured = await fetchFeaturedMarket();
-  const markets = await fetchMarkets({
-    page,
-    sortBy,
-    orderBy,
-    searchEmojis,
-  });
-  const priceFeed = await fetchPriceFeed({});
-
-  const geoblocked = await isUserGeoblocked(headers().get("x-real-ip"));
-
-  return (
-    <HomePageComponent
-      featured={featured}
-      markets={markets}
-      numRegisteredMarkets={numRegisteredMarkets}
-      page={page}
-      sortBy={sortBy}
-      searchBytes={q}
-      geoblocked={geoblocked}
-      priceFeed={priceFeed}
-    />
-  );
+export default function Home() {
+  return <LoadingComponent />;
 }
```

Comment:
What exactly is going on here? Did you change this to a loading page because it's just a redirect now?

Reply:
I think we talked about this. I'm redirecting all requests from `/` to `/home` so that we don't have a duplicated home page. This actually introduced an error where the `/home` was working properly but `/` not. I left a loading component here just in case this page somehow renders, but it should never happen.